openrtb_ext/response.go

```diff
@@ -96,3 +101,44 @@ const (
 	UserSyncIframe UserSyncType = "iframe"
 	UserSyncPixel  UserSyncType = "pixel"
 )
+
+// Bid is subset of Bid object with exact json signature
+// defined at https://github.com/prebid/openrtb/blob/v17.1.0/openrtb2/bid.go.
+// It also contains the custom fields
+type Bid struct {
```

Comment:
Looks great to me. Thank you. I would recommend to change the name to something which clearly associates it with a nonBid. You're welcomed to come up with something clever, but I'm fine with something a bit obvious, if not a little silly, like`nonBidBid`.

Reply:
I have renamed it to `NonBidObject`